### PR TITLE
Mobile Stories block (part 5): add empty placeholder

### PR DESCRIPTION
**Pull request by mzorz**
This PR builds on top of https://github.com/Automattic/jetpack/pull/17458

This PR just adds a proper MediaPlaceholder component for empty blocks, but also blocks the adding media flow from it for now using `pointerEvents:none` given we are handling the flow for adding media to a Story block differently, through an external component (the Story creator, dependent on the host app)

Related PRs:
Gutenberg-mobile https://github.com/wordpress-mobile/gutenberg-mobile/pull/2747
Jetpack https://github.com/Automattic/jetpack/pull/17539
WPANdroid https://github.com/wordpress-mobile/WordPress-Android/pull/13174

To test:
- use WPAndroid PR: WPAndroid https://github.com/wordpress-mobile/WordPress-Android/pull/13174

### CODE REVIEW

**Review by jd-alexander — COMMENTED** [review]
These changes look good @mzorz Let's await @etoledom as well to give a second opinion and approve if he sees fit.

File: extensions/blocks/story/edit.native.js
Comment (etoledom) [inline extensions/blocks/story/edit.native.js:241]: We can import `MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO` from `core/block-editor` to avoid hard typed strings

**Review by etoledom — COMMENTED** [review]
Looking pretty nice! 🎉 

I mostly have a couple of questions:

Something I'm not sure about, since there are no visuals on the description: Is this the expected design for the story block placeholder? Looks quite different from other media placeholders:

Screenshot 2020-10-26 at 20 37 50
 
Also, will the `ADD MEDIA` button do something at some point later on?

File: extensions/blocks/story/edit.native.js
Comment (mzorz) [reply extensions/blocks/story/edit.native.js:241]: 👍 tackled in deb66b9f38 🙇

**Comment by mzorz** [comment]
> Something I'm not sure about, since there are no visuals on the description: Is this the expected design for the story block placeholder? Looks quite different from other media placeholders

You're right - double checked this, will ensure it has the correct look & feel and get back here 👍

**Comment by mzorz** [comment]
Alright! Done the changes for the placeholder look here fc24451e31:

emptyblock2

**Review by etoledom — APPROVED** [review]
Thank you @mzorz for the update, nice job! 🎉

**Comment by mzorz** [comment]
Thank you for your review @etoledom ! 🙇 🎉 :D

**Pull request merged**
